### PR TITLE
cross function reformat

### PR DESCRIPTION
**Pull request by Quickbick**
cross #1971 

I have changes some wording in the docstring to better match the wording used in the Array API Standard and added the default nestable statement from the docstring examples to the cross docstring.

Many of the tasks that are stated in the reformat had already been done, so I'm somewhat concerned that the function was already supposed to be done or I'm looking past something I need to do.

### CODE REVIEW

**Comment by Quickbick** [comment]
So are the check issues something that's in the file I changed? Because from what I can tell they are not, unless I'm misunderstanding something.

**Review by Ishticode — APPROVED** [review]
Hi @Quickbick. Thank you for the PR 😊
Looks fine but not sure if its a reformatting task. I will ask Dan to have a look...

**Review by djl11 — CHANGES_REQUESTED** [review]
please add the required `ivy.Array` and `ivy.Container` instance methods, and their docstrings and docstring examples.

**Comment by Quickbick** [comment]
I missed those files, I will get them done!

**Comment by Quickbick** [comment]
That should be the files I missed. I updated the array implementation and added the container implementation. Ready for your team to take another look.

File: ivy/array/linear_algebra.py
Comment (djl11) [inline ivy/array/linear_algebra.py:78]: you should mention that the docstring has been borrowed from the Array API standard, as explained in the docstring section.

File: ivy/array/linear_algebra.py
Comment (djl11) [inline ivy/array/linear_algebra.py:133]: we shouldn't add any `ivy.NativeArray` examples.

File: ivy/array/linear_algebra.py
Comment (djl11) [inline ivy/array/linear_algebra.py:151]: should not be added here, please read deep dive

**Review by djl11 — CHANGES_REQUESTED** [review]
please read the deep dive carefully, particularly sections on docstrings and docstring examples, and then please add the required changes and request another review if you are feeling confident that the PR is correct.

**Comment by Quickbick** [comment]
I believe I found all the mistakes I had made with the docstrings and docstring examples.

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hi @Quickbick few changes required
The examples for static method under Container should create a Container first, and then call the static method only not instance method. 
Examples of containers should not be found under Array instance method.
Please take a look at the deep dive (docstring examples section) as Dan pointed out and I am happy to answer any further questions based on that 😊.

**Comment by Quickbick** [comment]
That should fix the docstring issues. I've read over the deep dive for docstring examples and am having a somewhat difficult time figuring out exactly what applies to my particular function. I think everything is taken care of, but I have been wrong on that account before with this project.

**Comment by Quickbick** [comment]
I also see some conflicts with the master branch now. It appears there was someone else working on the cross function. Should I look into resolving those or just wait until after you have looked at my work again?

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hi @Quickbick. Thank you for the review request.
Please kindly take a look at the instructions (especially [Docstring examples](https://lets-unify.ai/ivy/deep_dive/13_docstring_examples.html), [Container](https://lets-unify.ai/ivy/deep_dive/5_containers.html) and [Array](https://lets-unify.ai/ivy/deep_dive/4_arrays.html) sections) from the deep dive as pointed out few times above.
For example, the docstring in ivy funtional shouldn't have native array or array instance examples. In addition, examples with the axis argument should be added also which are not found. Please feel free to ask questions when taking a look at the guide and make changes accordingly and I will be happy to answer : ).

**Comment by Quickbick** [comment]
> Hi @Quickbick. Thank you for the review request. Please kindly take a look at the instructions (especially [Docstring examples](https://lets-unify.ai/ivy/deep_dive/13_docstring_examples.html), [Container](https://lets-unify.ai/ivy/deep_dive/5_containers.html) and [Array](https://lets-unify.ai/ivy/deep_dive/4_arrays.html) sections) from the deep dive as pointed out few times above. For example, the docstring in ivy funtional shouldn't have native array or array instance examples. In addition, examples with the axis argument should be added also which are not found. Please feel free to ask questions when taking a look at the guide and make changes accordingly and I will be happy to answer : ).

I realized my mistake with the instance methods in the functional section and fixed those as well as some other issues with my docstring via the deep dive examples. There is already an example in functional where the axis value is something besides default, do I need to have that for the container static, container instance, and array instance as well, or should it just be in the functional?

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hi. The examples need a revision. For example instance method calls **should not** be like `x.cross(x,y)`. Creating the inputs `x1` and `x2` for `static_cross` is calling `static_cross` for some reason. These errors are in multiple locations and I didn't make individual comments. Please kindly scan through and ask review after fixing.

File: ivy/container/linear_algebra.py
Comment (djl11) [inline ivy/container/linear_algebra.py:259]: can you indent the contents of the printed container by 4 spaces for all examples?

**Review by djl11 — CHANGES_REQUESTED** [review]
small changes then it's good to go I think, provided all the docstring unit tests are passing!

**Review by djl11 — CHANGES_REQUESTED** [review]
please fix merge conflicts, and then once we can see that the docstring tests are passing and @RickSanchezStoic approves then I think it's good to merge!

File: ivy/functional/ivy/linear_algebra.py
Comment (djl11) [inline ivy/functional/ivy/linear_algebra.py:187]: "This docstring is an extension of the..." should not have been removed. Check [here](https://lets-unify.ai/ivy/deep_dive/12_docstrings.html).

File: ivy/array/linear_algebra.py
Comment (djl11) [inline ivy/array/linear_algebra.py:118]: this does not need to be added for array instance methods, check [here](https://lets-unify.ai/ivy/deep_dive/12_docstrings.html).

**Review by djl11 — CHANGES_REQUESTED** [review]

**Comment by djl11** [comment]
So actually this function was already reformatted, sorry for the confusion. Given the time you've put into this PR, if it's part of an application then we'll progress you to the next stage regardless. Thanks @Quickbick.

**Pull request closed without merging**
